Reference/Querying/IPublishedContent/ExtensionMethods.md

```diff
@@ -0,0 +1,4 @@
+# IPublishedContent Extension Methods
+
+### .GetKey() (version 7.6+)
```

Comment:
I would not add the version on the same line as that might collide with the parameters. 😁